tools/test.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+# Runs the full test suite.
+# Usage: ./tools/test.py out/Debug
```

Comment:
Strangely `python tools/test.py out/Debug` fails for me, but `python tools/test.py out\Debug` works.
(Note the forward slash vs backslash)

I don't like that it's so sensitive to forward vs backward slashes.
Other tools don't have that. I can run `ninja -C out/Debug` and `gn gen out/Debug` with no issue.